docs/benchmark.rst

```diff
@@ -167,4 +167,11 @@ Here are the detailed parsing performance results for various models:
      - 0.451
      - 0.257
      - 4.54
-     - 0
```

Comment:
Minor: 
Q: Do we have a section in the doc explaining our thought process in relation to router priority?